src/Lykke.Service.EthereumCore/Modules/ControllerModule.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Linq;
+using System.Reflection;
+using Autofac;
+using Autofac.Extensions.DependencyInjection;
+using Autofac.Features.AttributeFilters;
+using Common.Log;
+using Lykke.Service.EthereumCore.AzureRepositories;
+using Lykke.Service.EthereumCore.Core.Services;
+using Lykke.Service.EthereumCore.Core.Settings;
+using Lykke.Service.EthereumCore.Core.Utils;
+using Lykke.Service.EthereumCore.Services;
+using Lykke.Service.RabbitMQ;
+using Lykke.SettingsReader;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.DependencyInjection;
+using MoreLinq;
+using Module = Autofac.Module;
+
+namespace Lykke.Service.EthereumCore.Modules
+{
+    public class ControllerModule : Module
```

Comment:
As far as I see the purpose of this module is to register Controllers but there is no need in registering controllers manually.

Reply:
You have to register controllers manually if you want to use Named and Keyed Services all across your app (Autofac feature)